packages/js/src/Modules/Verto/index.ts

```diff
@@ -21,7 +21,68 @@ export default class Verto extends BrowserSession {
     return isValidOptions(this.options);
   }
 
+  /**
+   * Makes a new outbound call.
+   *
+   * This method receives an object `options` with the following properties:
+   *
+   * @param options Options object for a new call.
+   * @param options.destinationNumber Phone number or SIP URI to dial.
+   * @param options.callerNumber Number to use as the caller ID when dialing out to a destination.
+   * @param options.callerName Name to use as the caller ID name when dialing out to a destination.
+   * @param options.id The identifier of the call.
+   * @param options.localStream If set, the call will use this stream instead of retrieving a new one.
+   * @param options.localElement Overrides client's default `localElement`.
+   * @param options.remoteElement Overrides client's default `remoteElement`.
+   * @param options.iceServers Overrides client's default `iceServers`.
+   * @param options.audio Overrides client's default audio constraints. Defaults to `true`
+   * @param options.video Overrides client's default video constraints. Defaults to `false`
+   * @param options.useStereo Uses stereo audio instead of mono.
+   * @param options.micId `deviceId` to use as microphone. Overrides the client's default one.
+   * @param options.camId `deviceId` to use as webcam. Overrides the client's default one.
+   * @param options.speakerId `deviceId` to use as speaker. Overrides the client's default one.
+   * @param options.onNotification Overrides client's default `telnyx.notification` handler for this call.
+   *
+   * @return `Promise<Call>` A promise fulfilled with the new outbound `Call` object
+   * or rejected with the error.
+   *
+   * @examples
+   *
+   * Making an outbound call to `+1 856-444-0362` using default values from the client:
```

Comment:
The description here states that the examples are for making an OB call to `+1 856-444-0362`, but the following examples are for empty options, which is confusing. Maybe a separate "Errors" section in examples?